src/dredd.coffee

```diff
@@ -41,6 +42,24 @@ class Dredd
     @transactions = []
     @runner = new Runner(@configuration)
     configureReporters(@configuration, @stats, @tests, @runner)
+    @logProxySettings()
+
+  logProxySettings: ->
+    proxySettings = []
+    for envVariable in Object.keys(process.env)
+      continue unless envVariable.toUpperCase() in PROXY_ENV_VARIABLES
```

Comment:
I saw a `caseless` library for checking the string, so it could be used here as well.